modules/monitoring-resizing-a-persistent-storage-volume.adoc

```diff
@@ -0,0 +1,192 @@
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+.. Add a new storage size for the PVC configuration for the component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    <component>:
+      volumeClaimTemplate:
+        spec:
+          resources:
+            requests:
+              storage: <amount_of_storage> <1>
+----
+<1> Specify the new size for the storage volume.
++
+The following example configures a PVC that changes the local persistent storage to 100 gigabytes for the Prometheus instance that monitors core {product-title} components:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes a PVC that changes local persistent storage for Alertmanager to 40 gigabytes:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: 40Gi
+----
+
+** *To configure a PVC for a component that monitors user-defined projects*:
+.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
+
+.. Update the PVC configuration for the monitoring component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    <component>:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class>
+          resources:
+            requests:
+              storage: <amount_of_storage> <1>
+----
+<1> Specify the new size for the storage volume.
++
+The following example changes the PVC size to 100 gigabytes the Prometheus instance that monitors user-defined projects:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes the PVC size to 20 gigabytes for Thanos Ruler:
```

Comment:
Just would like to mention that users could also resize the size of Alermanager and Prometheus for UWM, not sure if we want to add examples  for that (personally I don't see the need) but just mentioning it

Reply:
I added a note about this at the beginning of this section of the procedure